File: JS/URI/1013.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var leia = lines[0];

var valores = leia.split(' ');

var maiorAb = (parseInt(valores[0])+parseInt(valores[1])+Math.abs(parseInt(valores[0])-parseInt(valores[1])))/2;
var maior = (maiorAb+parseInt(valores[2])+Math.abs(maiorAb-parseInt(valores[2])))/2;

console.log(maior + " eh o maior");

File: Python/URI/1013.py
valor = input().split(" ")

a, b, c = valor

maiorAb = (int(a)+int(b)+abs(int(a)-int(b)))/2
maior = (maiorAb+int(c)+abs(maiorAb-int(c)))/2

print(int(maior), "eh o maior")